css/styles.css

```diff
@@ -0,0 +1,148 @@
+* {
+  box-sizing: border-box;
+}
+
+article {
+  background-image: url("../images/coworking.jpg");
+  background-repeat: no-repeat;
+  background-position: center top;
+  height: 400px;
+  border: 1px solid #333;
+  width: 100%;
+  float: left;
+}
+
+body {
```

Comment:
The rules for body should be at the top of your css file since everything inherits from its parent container.  And in your project that is the body tag. 🔨